sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -52,8 +51,10 @@ export { Delivery }
 export interface GetMessageIteratorOptions extends OperationOptions, WaitTimeOptions {
 }
 
+// Warning: (ae-forgotten-export) The symbol "GetReceiverOptions" needs to be exported by the entry point index.d.ts
```

Comment:
export `GetReceiverOptions`.


Question: `GetReceiverOptions` and `GetSenderOptions` - are these names fine?

Reply:
Those are consistent with what we do - "name of function"Options